test/Interop/Cxx/class/copy-module-interface.swift

```diff
@@ -0,0 +1,5 @@
+// RUN: %target-swift-ide-test -print-module -module-to-print=Constructors -I %S/Inputs -source-filename=x -enable-cxx-interop | %FileCheck %s
```

Comment:
Please rename this swift file to start with 'constructors-' to match the existing naming pattern. `constructors-copy-module-interface.swift`.